PERL_MODULES

```diff
@@ -14,3 +14,7 @@ IO::Pty
 LWP
 Authen::Radius
 Path::Tiny
+InfluxDB::HTTP
+InfluxDB::LineProtocol
```

Comment:
is this a standalone module, or does it require some influxdb library to be present?

Reply:
Apart from the modules (and their perl dependencies) there are no other requirements. Communication is done via HTTP POST to the influxdb server. LineProtocol just formats the data in the InfluxDB specification, and the HTTP module is probably a wrapper for LWP, but I haven't looked into it.